docs/.vuepress/config/sidebar-developer.js

```diff
@@ -509,7 +510,7 @@ const developer = [
         'New WYSIWYG field in admin panel',
       ],
       // ['/developer-docs/latest/guides/client', 'Setup a third party client'],
-      ['/developer-docs/latest/guides/unit-testing', 'Unit testing'],
+      //['/developer-docs/latest/guides/unit-testing', 'Unit testing'],
```

Comment:
Thanks, Shaun. LGTM except that here, instead of commenting the line, I think you can simply delete it.